src/dfmt/formatter.d

```diff
@@ -745,10 +749,13 @@ private:
         }
         else if (peekBackIs(tok!"identifier") && (peekBack2Is(tok!"{", true)
                 || peekBack2Is(tok!"}", true) || peekBack2Is(tok!";", true)
-                || peekBack2Is(tok!":", true)) && !(isBlockHeader(1) && !peekIs(tok!"if")))
+                || peekBack2Is(tok!":", true) || peekBack2Is(tok!",", true))
+                && !(isBlockHeader(1) && !peekIs(tok!"if")))
```

Comment:
this `&&` looks very confusing now, can you surround the 3 expressions with parentheses so they look like they belong together again?

Reply:
What you mentioned is peekBack2Is(tok!",", true)), isBlockHeader(1) and peekIs(tok!"if")?

I cannot surround these because the first one is already surrounded with the previous terms.

Reply:
```d
if (...
		peekBack2Is(tok!":", true) || /* add parens here: */ (peekBack2Is(tok!",", true))
		&& !(isBlockHeader(1) && !peekIs(tok!"if")))
```

Reply:
sorry - my issue, I misread it because it had too many parentheses already lol

Reply:
What about using staticMap instead?
```d
staticMap!(tok, "{","}",";",":",",")].any!((ulong token) => peekBack2Is(cast(IdType)token, true))
```

Reply:
that actually looks really readable then, but is the cast there needed? Maybe just using an array of `tok!""` wouldn't harm either and it would drop some work from the compiler. (because staticMap might be a bit of unnecessary compiler overhead here)

Reply:
I tried but I could not remove the cast from this code because the type of `tok`  is `ulong` and `IdType` cannot be `ulong`.
This is the same wheter or not I use staticMap.